lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -431,6 +430,7 @@ def test__has_access_descriptor(self):
         user = Mock()
         descriptor = Mock(user_partitions=[])
         descriptor._class_tags = {}
+        descriptor.merged_group_access = {}
```

Comment:
Note to reviewers: needed because of the removal of the short circuit logic in access.py.